modules/ossm-release-notes-3-1-new-features-enhancements.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/ossm-release-notes/ossm-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-release-3-1-new-features-enhancements_{context}"]
+= {SMProductName} version 3.1 new features and enhancements
+
+This release makes {SMProductName} 3.1 generally available, adds new features, addresses Common Vulnerabilities and Exposures (CVEs), and is supported on {ocp-product-title} 4.14 and later.
+
+For a list of supported component versions and support features, see "Service Mesh 3.0 feature support tables".
+
```

Comment:
Can you delete the following comment from the source file or is it worth retaining?